content/en/docs/contribute/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+content_template: templates/concept
+title: Contribute to Kubernetes docs
+linktitle: Contribute
+main_menu: true
+weight: 80
+---
+
+{{% capture overview %}}
+
+If you're reading this page, you might be interested in contributing to the
+Kubernetes documentation or website, and we are glad you're here! Anyone can
+contribute, whether you're new to the project or you've been around a long time,
+and whether you self-identify as a developer, an end user, or someone who just
+can't stand seeing typos.
+
+Looking for the [style guide](/docs/contribute/style/style-guide/)?
```

Comment:
Instead of "Looking for the"   Perhaps say, Before you start contributing we highly recommend you review our [style guide] .  This will help to get your pull requests merged more quickly.